### DISCUSSION TITLE
A description problem from the document

### DISCUSSION THREAD

**Discussion by zms9110750** [dotnet/csharplang#6196] [General] [answered]
In the Chinese language, "constructor" is translated as "construction function".
But conceptually, c# should have only methods but no functions.
(Interestingly, the original English text has neither "method" nor "function")
Can this problem be fixed?
("Destructors "are also translated into "destructors function". "Finalizers" only "Finalizers")

**Comment by 333fred** [comment, marked as answer]
What document are you referring to? Translation bugs should be filled directly with that repo. Also note that C# does have functions: local functions.

**Reply by zms9110750** [reply]
On these two pages：
https://docs.microsoft.com/zh-cn/dotnet/csharp/programming-guide/classes-and-structs/using-constructors
https://docs.microsoft.com/zh-cn/dotnet/csharp/programming-guide/classes-and-structs/finalizers
I can see page editing options when reading in English, but not when reading in Chinese.
I don't quite understand the word "repo" you use（Unable to translate words that fit the context）. Do you mean the "feedback" button at the bottom of the page in English?

**Reply by zms9110750** [reply]
So, methods only describe members of a class? Local function is not a method because it is not a member of a class?

**Reply by 333fred** [reply]
> Do you mean the "feedback" button at the bottom of the page in English?

Yes, that will bring you to the appropriate repo for that docs page.

> So, methods only describe members of a class? Local function is not a method because it is not a member of a class?

That's correct.
